src/LendingPool.sol

```diff
@@ -517,6 +517,30 @@ contract LendingPool is Guardian, TrustedCreditor, DebtToken, InterestRateModule
         }
     }
 
+    /**
+     * @notice Skims any surplus funds in the LendingPool to the treasury
+     * @dev In a normal conditions (when there are no ongoing auctions), the total Claimable Liquidity should be equal
+     * to the sum of the available funds (the balanceOf() the underlying asset) in the pool and the total open debt.
+     * Due to rounding errors of the ERC4626 calculations (rounding will always cause the actual Claimable Liquidity to be bigger than totalRealisedLiquidity)
```

Comment:
clarify that this is: "the total amount of assets in the pool after everyone repaid their debts and all interests are synced vs the number in totalRealisedLiquidity"